src/App.tsx

```diff
@@ -1,8 +1,38 @@
 import React from 'react';
 import { Login } from './views/Login';
+import { Navigate, Route, Routes } from 'react-router-dom';
+import { Dashboard } from 'views/Dashboard';
+import { Leads } from 'views/Leads';
+import { RequireAuth, useIsAuthenticated } from 'react-auth-kit';
 
 function App() {
-   return <Login />;
+   const isAuthenticated = useIsAuthenticated();
+
+   return (
+      <Routes>
+         <Route path="/" element={<Navigate to="/login" />} />
+         <Route
+            path="/login"
+            element={!isAuthenticated() ? <Login /> : <Navigate to="/dashboard" />}
+         />
+         <Route
+            path="/dashboard"
+            element={
+               <RequireAuth loginPath="/login">
```

Comment:
**nitpick:** You are duplicating route paths as strings. Let's do some const or enum for keeping route values, and then reuse it

Reply:
@Sumick Done :)